Source/WebCore/rendering/GridMasonryLayout.cpp

```diff
@@ -199,6 +199,9 @@ void GridMasonryLayout::insertIntoGridAndLayoutItem(const GridTrackSizingAlgorit
         if (!(gridItem.style().logicalWidth().isAuto() || gridItem.style().logicalWidth().isPercent()))
             return false;
 
+        if (gridItem.style().writingMode().isOrthogonal(m_renderGrid.style().writingMode()))
```

Comment:
This seems to include all orthogonal items not just ones in masonry columns. Is this intended? Does this need to change or the commit message need to change? 

I would imagine we would need to compute the min/max content size of an orthogonal masonry item in a masonry rows grid since we would not have any inline constraints for it after track sizing.

Reply:
This check will only be hit when we masonry columns is set. This is due to the check above comparing the layout phase. The only time layout phase is set to min or max content is in computeIntrinsicLogicalWidths where we already have the same check.

I added a debug assert to verify this assumption holds.